lib/iris/experimental/ugrid/mesh.py

```diff
@@ -1044,7 +1044,19 @@ def line(text, i_indent=0):
                 else:
                     show = val is not None
                 if show:
-                    line(f"{name}: {val!r}", 1)
+                    if name == "attributes":
+                        # Use a multi-line form for this.
+                        line("attributes:", 1)
+                        max_attname_len = max(len(attr) for attr in val.keys())
+                        for attrname, attrval in val.items():
+                            attrname = attrname.ljust(max_attname_len)
+                            if isinstance(attrval, str):
+                                # quote strings
+                                attrval = repr(attrval)
+                            attr_string = f"{attrname}  {attrval}"
```

Comment:
This is essentially the same as the code in _DimensionalMetadata.summary, [above](https://github.com/SciTools/iris/pull/4528/files?file-filters%5B%5D=.py&show-viewed-files=true#r792715108).

So there is potentially an option to combine these approaches, and possibly even with that in CubeSummary [above#2](https://github.com/SciTools/iris/pull/4528/files?file-filters%5B%5D=.py&show-viewed-files=true#r792717644).

It is also notable that we ***could*** be clipping over-long strings here, as Cube.summary does.
Not sure if that is really needed, but if we combine them it would make sense.